packages/examples/cvat/exchange-oracle/src/core/config.py

```diff
@@ -195,6 +195,9 @@ class CvatConfig:
     cvat_iou_threshold = float(os.environ.get("CVAT_IOU_THRESHOLD", 0.8))
     cvat_oks_sigma = float(os.environ.get("CVAT_OKS_SIGMA", 0.1))
 
+    cvat_polygons_iou_threshold = float(os.environ.get("CVAT_POLYGONS_IOU_THRESHOLD", 0.5))
```

Comment:
Please update `.env.template` with the new variable

Reply:
Done